Changed Topology view to Graph view as text wherever required. (#29290)
openshift/openshift-docs: modules/odc-interacting-with-applications-and-components.adoc

[id="odc-interacting-with-applications-and-components_{context}"]
= Interacting with applications and components

> The *Topology* view in the *Developer* perspective of the web console provides the following options to interact with applications and components:

Reviewer: ```suggestion
The *Topology* page in the *Developer* perspective of the web console provides the following options to interact with applications and components:
```

> * Use the *Shortcuts* menu on the upper-right of the screen to navigate components in the *Graph view*.

Reviewer 2: Hmmm... the Shortcuts menu is NEXT to the icon, it's not the name of the icon. it's a different interaction point.

Maybe something like:
```suggestion
* Use the icon (<insert the icon here>) in the upper-right of the screen to navigate components in the *Graph view*.
```

+
[NOTE]
====

> Serverless applications take some time to load and display on the *Graph view*. When you create a serverless application, it first creates a service resource and then a revision. After that, it is deployed and displayed on the *Graph view*. If it is the only workload, you might be redirected to the *Add* page. After the revision is deployed, the serverless application is displayed on the *Graph view*.

Reviewer: ```suggestion
Serverless applications take some time to load and display on the *Graph view*. When you deploy a serverless application, it first creates a service resource and then a revision. After that, it is deployed and displayed on the *Graph view*. If it is the only workload, you might be redirected to the *Add* page. After the revision is deployed, the serverless application is displayed on the *Graph view*.
```
